packages/helper/src/utils/Utils.ts

```diff
@@ -165,22 +165,22 @@ export function xml2js(messageString: string): MosModel.AnyXMLObject {
 			}
 		}
 	}
-	concatChildrenAndTraverseObject(object)
+	// concatChildrenAndTraverseObject(object)
```

Comment:
This looks like an accidental change, and I suspect is the cause of the test failures

Reply:
ah you are correct of course!